docs/contributing.md

```diff
@@ -35,4 +31,5 @@ information on using pull requests.
 This project follows [Google's Open Source Community
 Guidelines](https://opensource.google/conduct/).
 
-[developer guide]: development.md
\ No newline at end of file
+[developer guide]: development.md
+[design document]: design-docs/00-template.md
```

Comment:
If we want to use the same template as `kpt`, perhaps we could point directly to the template in the `kpt` repo so that its only defined in one place. WDYT?

Reply:
I think the design changes in kpt are not the same as ConfigSync.  I am open to decoupling the approvers but it seems like we are starting with combined group first.

I vote for design changes to ConfigSync being in the same repo as the code.

Reply:
I'm only suggesting that we point to the same template, I agree that config sync design docs themselves should be committed to the config sync repo

Reply:
I think cross repo template pointing is probably a little bit strange.  Once we get one design doc PR I am actually going to change the reference to the local repo for a PR example too.